src/diffpy/utils/diffraction_objects.py

```diff
@@ -422,11 +424,16 @@ def scale_to(self, target_diff_object, q=None, tth=None, d=None, offset=0):
         """
         scaled = self.copy()
         count = sum([q is not None, tth is not None, d is not None])
-        if count != 1:
+        if count > 1:
             raise ValueError(
-                "You must specify exactly one of 'q', 'tth', or 'd'. Please rerun specifying only one."
+                "You must specify none or exactly one of 'q', 'tth', or 'd'. "
+                "Please provide either none or one value."
             )
 
+        if count == 0:
+            scaled._all_arrays[:, 0] *= max(target_diff_object.on_q()[1]) / max(self.on_q()[1])
+            return scaled
```

Comment:
when no xvalue was specified

Reply:
If user specified an offset do we want to add offset to the scaled array?

Reply:
yes, that is correct.  Add the offset to the y-array after scaling

Reply:
added the offset here